src/xarray_regrid/methods/conservative.py

```diff
@@ -79,6 +79,8 @@ def conservative_regrid_dataset(
     """Dataset implementation of the conservative regridding method."""
     data_vars: list[str] = list(data.data_vars)
     dataarrays = [data[var] for var in data_vars]
+    attrs = data.attrs
+    da_attrs = [da.attrs for da in dataarrays]
```

Comment:
This works great for the base attributes and attributes of variables. However, any coordinates' attributes should be retained as well.

Maybe update these lines to be something like:
```suggestion
    data_vars: list[str] = list(data.data_vars)
    data_coords: list[str] = list(data.coords)
    dataarrays = [data[var] for var in data_vars]
    datacoords = [data[var] for var in data_coords]
    attrs = data.attrs
    da_attrs = [da.attrs for da in dataarrays]
    dcrds_attrs = [dcrds.attrs for dcrds in datacoords]
```

Reply:
Ah good point, I forgot these!